### DISCUSSION TITLE
能设置http回调吗？怎么注册一个http回调地址，并指定端口呢

### DISCUSSION THREAD

**Discussion by songbj32** [ttttupup/wxhelper#461] [Q&A]
老师，怎么注册一个http回调地址，并指定端口的

**Comment by ttttupup** [comment]
wechat目录下config.ini 配置文件的默认配置
// HiddenDll ： 0.不隐藏注入dll 1.隐藏dll
// HttpServerPort ：dll自身提供的http服务的端口
// HttpServerHost ：dll自身提供的http服务的监听地址
// RecvMessageMode：开启消息hook时使用接收消息的方式. tcp | http
// tcp模式下通过tcp连接发送消息,http模式下通过http发送消息
// RecvTcpIp ：接收消息绑定的ip
// RecvTcpPort：tcp模式下绑定的端口
// RecvHttpUrl：http模式下推送的接口地址
// RecvHttpTimeout： http模式下的超时时间

HiddenDll=0,
HttpServerPort=80,
HttpServerHost=http://0.0.0.0/,
RecvMessageMode=tcp,
RecvTcpIp=127.0.0.1,
RecvTcpPort=19099,
RecvHttpUrl=127.0.0.1,
RecvHttpTimeout=3000,
